### PR TITLE
Moving away from Nancy

### PR DESCRIPTION
**Pull request by cniackz**
### Objective:

Moving away from `Nancy` to use `vulncheck`

### Story:

`Nancy` test is failing and the issue is recent: https://github.com/sonatype-nexus-community/nancy/issues/263

image

### Additional Info:

https://pkg.go.dev/golang.org/x/vuln/vulncheck

### CODE REVIEW

**Review by Alevsk — APPROVED** [review]
Are we adding the vulncheck job in a different PR ?

**Review by pjuarezd — DISMISSED** [review]

**Comment by harshavardhana** [comment]
Workflow is not running 

> The workflow is not valid. .github/workflows/jobs.yaml (Line: 26, Col: 9): Job 'replication' depends on unknown job 'vulnerable-dependencies-checks'. .github/workflows/jobs.yaml (Line: 94, Col: 9): Job 'sso-integration' depends on unknown job 'vulnerable-dependencies-checks'.

@cniackz

**Comment by cniackz** [comment]
no, i am adding it vulncheck here @Alevsk & this time the workflow should work, thx @harshavardhana

**Review by reivaj05 — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by dvaldivia — APPROVED** [review]

File: .github/workflows/jobs.yaml [hunk 1]
```diff
@@ -250,31 +250,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [ 1.18.x ]
-        os: [ ubuntu-latest ]
+        go-version: [ 1.19 ]
```
Comment (harshavardhana) [inline .github/workflows/jobs.yaml:253]:
```suggestion
        go-version: [ 1.19.x ]
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: .github/workflows/jobs.yaml (hunk 1 above)
Comment (pjuarezd) [reply .github/workflows/jobs.yaml:253]: Last time I’ve tried that made no effect, the GitHub action was still using the latest cached version of go. 

The `check-latest: true` did the trick of get the latest revision.

anyways, having it doesn’t hurt

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**